tslint.json

```diff
@@ -1,6 +1,6 @@
 {
   "rules": {
-    "indent": [true, "tabs"],
+    "indent": [true, "spaces"],
     "semicolon": [true, "always"]
```

Comment:
Given that we don't use semicolons anywhere else, we should probably bring this rule inline

Reply:
There's a few options then ( and I am a little ignorant of typescript to being with ) but from what you're saying, this should be changed to "never":

```
One of the following arguments must be provided:

    "always" enforces semicolons at the end of every statement.
    "never" disallows semicolons at the end of every statement except for when they are necessary.

The following arguments may be optionally provided:

    "ignore-interfaces" skips checking semicolons at the end of interface members.
    "ignore-bound-class-methods" skips checking semicolons at the end of bound class methods.
    "strict-bound-class-methods" disables any special handling of bound class methods and treats them as any other assignment. This option overrides "ignore-bound-class-methods".
```

https://palantir.github.io/tslint/rules/semicolon/

Reply:
Yup, I'm suggesting `never` 👍 But again, I've never touched this code, so I may be missing some context

Reply:
But we can do that in a different PR

Reply:
I had no idea that I was in the midst of semi-colon haters ;)  I don't mind semicolons ... it's that "except when they are necessary" that always gets ya. Generally, whatever we do should be consistent, automatically (GH action) linted and ideally auto-formatted via `pretter` etc.